benchmarking/chain-sync/benchmark-chain-sync-mainnet.sh

```diff
@@ -32,6 +32,8 @@ exec timeout $CPU_TIME_LIMIT ${NODE} \
   --tracing-verbosity-maximal \
   --trace-mempool \
   --trace-forge \
+  --trace-block-fetch-decisions \
+  --trace-chain-db \
```

Comment:
Can we do exactly the same for the `cardano-node/scripts/mainnet.sh` to fix it there?

Reply:
Yes, but that would risk triggering trace/monitoring related space leak.

Reply:
if that script does not use the "LiveView" then we do not have the tracers added.

Reply:
The script uses liveview. The leak seems to be specifically triggered by the `--trace-chain-db` option, which is currently missing.
I'm going to wait with merging this PR and address mainnet.sh until that issue is fixed.